tools/ktest-tool/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#===------------------------------------------------------------------------===#
+#
+#                     The KLEE Symbolic Virtual Machine
+#
+# This file is distributed under the University of Illinois Open Source
+# License. See LICENSE.TXT for details.
+#
+#===------------------------------------------------------------------------===#
+install(PROGRAMS ktest-tool DESTINATION bin)
+
+# Copy into the build directory's binary directory
+# so integration tests can find it
+configure_file(ktest-tool "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/ktest-tool" COPYONLY)
```

Comment:
@delcypher , why do you need both install(...) and this line?  Shouldn't the install line copy the tool in `bin`?  This need to be better documented.  It might be useful to add some notes in the Developer's Guide about the steps needed say, to add a new tool, or use the tool in the tests.